mkdocs/pages/en/devbook/transactions/messages.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Messages
+---
+
+# Sending Messages with Transfer Transactions
+
+<Transfer transactions:> can include an optional message field, which allows attaching up to 1,024 bytes of data to the
+transaction.
+Messages can be sent as plain text or encrypted using the recipient's public key, ensuring only the intended recipient
+can read them.
+
+This tutorial shows how to send both plain and encrypted messages, and demonstrates how to decrypt received messages.
+
+## Prerequisites
+
+If you have not done so already, start with the [Hello World](../start/hello-world.md) tutorial to make sure your
+development environment is set up correctly.
+
+Additionally, check the [Transfer transaction](transfer.md) tutorial to understand how fee
+calculation, network time, and transaction confirmation works.
+
+## Full Code
+
+{% import 'tutorial.jinja2' as tutorial with context %}
+
+{{ tutorial.code_full('devbook/transactions/messages', ['py', 'js']) }}
+
+## Code Explanation
+
+### Setting Up Accounts
+
+{{ tutorial.code_snippet(['py:14:43', 'js:10:32']) }}
+
+The tutorial uses two accounts: a sender and a recipient.
+The code reads the sender's private key from the `PRIVATE_KEY` environment variable and the recipient's public key from
+the `RECIPIENT_PUBLIC_KEY` environment variable.
+Both variables default to test keys if not set.
+
+In a real application, you typically have the recipient's address rather than their public key.
+To encrypt messages, you need to obtain their public key by querying the network using the <get:/accounts/{accountId}>
+endpoint.
+The public key is only available for accounts that have previously sent transactions.
+
+### Sending a Plain Text Message
+
+{{ tutorial.code_snippet(['py:45:108', 'js:34:91']) }}
```

Comment:
This snippet is way too long. We don't need to show all the code, just the parts relevant to the topic of the tutorial.

You can leave out the parts about fetching network time and fees, and in the text you can send readers to the Transfer tutorial.
Same thing with the part about announcing the tx, that's also explained in the Transfer tutorial.
So the snippet should only be lines 57 to 74 (in the JS file)